src/utils/upload/processUpload.js

```diff
@@ -66,13 +66,20 @@ const compressAndUploadSingleFile = async (
 ) => {
   let loadedFile = null;
 
+  const filePropertiesToUpdate = {
+    size: file.size,
+    fileObject: file.fileObject,
+  };
+
   try {
     loadedFile = await loadAndCompressIfNecessary(file, () => (
       dispatch(
         updateSampleFile(
           sampleUuid,
           fileName,
-          { upload: { status: UploadStatus.COMPRESSING } },
+          {
```

Comment:
super nitpick: compress again this and other line

Reply:
changed